ocean_lib/data_provider/test/test_data_service_provider.py

```diff
@@ -243,13 +243,15 @@ def test_fileinfo(
     )
     assert fileinfo_result.status_code == 200
     files_info = fileinfo_result.json()
-    assert len(files_info) == 2
+
+    assert len(files_info) == 3
```

Comment:
I would suggest something like:
```
expected_content = ["text/plain", "text/xml", "application/octet-stream"]
````
and then `assert file["contentType"] = expected_content[file_index]`

how about that?